stdlib/contrib/sranka/opsgenie/opsgenie.flux

```diff
@@ -1,26 +1,73 @@
+// Package opsgenie provides functions that send alerts to
+// [Atlassian Opsgenie](https://www.atlassian.com/software/opsgenie)
+// using the [Opsgenie v2 API](https://docs.opsgenie.com/docs/alert-api#create-alert).
+//
+// introduced: 0.84.0
 package opsgenie
 
 
 import "http"
 import "json"
 import "strings"
 
-// respondersToJSON converts an array of responder strings to JSON array that can be embedded into an alert message
+// respondersToJSON converts an array of Opsgenie responder strings
+// to a string-encoded JSON array that can be embedded in an alert message.
+//
+// ## Parameters
+// - v: (Required) Array of Opsgenie responder strings.
+//   Responder strings must begin with
+//   `user: `, `team: `, `escalation: `, or `schedule: `.
```

Comment:
~~~suggestion
// - v: Array of Opsgenie responder strings.
//
//   Responder strings must begin with
//   `user: `, `team: `, `escalation: `, or `schedule: `.
//
// ## Examples
// ### Convert strings to Opsgenie responder strings
// ```
// import "contrib/sranka/opsgenie"
//
// opsgenie.respondersToJSON(
//     v: [
//         "user: example-user",
//         "team: example-team",
//         "escalation: example-escalation",
//         "schedule: example-schedule",
//     ],
// )
//
// // Returns the following JSON string:
// // [{"type":"user","username":" example-user"},{"type":"team","name":" example-team"},{"type":"escalation","name":" example-escalation"},{"type":"schedule","name":" example-schedule"}]
// ```
~~~